cmd/svcat/plan/get_cmd.go

```diff
@@ -55,6 +58,13 @@ func NewGetCmd(cxt *command.Context) *cobra.Command {
 		false,
 		"Whether or not to get the plan by UUID (the default is by name)",
 	)
+	cmd.Flags().StringVarP(
+		&getCmd.classFilter,
+		"class",
+		"c",
+		"",
+		"Whether or not to filter the plan based on class (the default is no filter)",
```

Comment:
Since `--class` is a string flag, and not a boolean, the description should be slightly different (to make it clear that it expects a name value and isn't a bool flag like `uuid`). Perhaps something like "Filter plans by the specified class".